src/fosslight_util/write_excel.py

```diff
@@ -82,18 +82,29 @@ def remove_empty_sheet(sheet_items):
     return success, final_sheet_to_print
 
 
-def get_header_row(sheet_name, sheet_content):
+def get_header_row(sheet_name, sheet_content, extended_header):
     selected_header = []
+
+    if extended_header:
+        for header_key in extended_header.keys():
+            _HEADER[header_key] = extended_header[header_key]
+
     for header_key in _HEADER.keys():
```

Comment:
I suggest the following.
- AS-IS: 
```
for header_key in _HEADER.keys():
        if header_key in sheet_name:
        if header_key == sheet_name:
            selected_header = _HEADER[header_key]
            break
```
- TO-BE:
```
selected_header = _HEADER[sheet_name]
```

Reply:
Removed if statement and change how to assign selected_header as suggested.